projects/1.18/assets/italian-delights-add-on-for-farmers-delight/italian_delight/lang/zh_cn.json

```
@@ -13,38 +17,66 @@
   "block.italian_delight.ham_pizza_placed": "火腿披萨",
   "block.italian_delight.muchroom_pizza_block": "蘑菇披萨",
+  "block.italian_delight.olive_door": "橄榄木门",
+  "block.italian_delight.olive_leaves": "橄榄树叶",
+  "block.italian_delight.olive_log": "橄榄木",
+  "block.italian_delight.olive_planks": "橄榄木板",
+  "block.italian_delight.olive_sapling": "橄榄树苗",
+  "block.italian_delight.olive_trapdoor": "橄榄木活板门",
+  "block.italian_delight.pandoro_block": "潘多酪",
+  "block.italian_delight.panettone_block": "潘娜托尼",
   "block.italian_delight.pizza_diavola_placed": "魔鬼披萨",
   "block.italian_delight.pizza_margherita": "玛格丽塔披萨",
   "block.italian_delight.pizza_margherita_1quarter": "玛格丽塔披萨",
   "block.italian_delight.woode_oven": "木烤炉",
+  "item.italian_delight.berrieddelight": "莓葬光明",
+  "item.italian_delight.dantes_spacial": "但丁特制披萨",
+  "item.italian_delight.dantes_special_slice": "但丁特制披萨切片",
   "item.italian_delight.dragonborn_wine": "龙裔酒",
   "item.italian_delight.glowberry_wine": "发光浆果酒",
   "item.italian_delight.grape_bunch": "葡萄串",
   "item.italian_delight.grape_seeds": "葡萄种子",
   "item.italian_delight.ham_pizza": "火腿披萨",
   "item.italian_delight.hasting_potion": "急迫之酒",
   "item.italian_delight.hellhound_wine": "地狱恶犬酒",
-  "item.italian_delight.iron_pizza_peel": "铁披萨刀",
+  "item.italian_delight.iron_pizza_peel": "披萨铲",
+  "item.italian_delight.mozzarella_panini": "马苏里拉帕尼尼",
+  "item.italian_delight.mozzarella_salad": "马苏里拉沙拉",
+  "item.italian_delight.mozzarella_slices": "马苏里拉切片",
+  "item.italian_delight.mozzarella": "马苏里拉",
   "item.italian_delight.mushroom_pizza": "蘑菇披萨",
   "item.italian_delight.mushroom_pizza_slice": "蘑菇披萨切片",
   "item.italian_delight.mushroom_risotto": "蘑菇烩饭",
+  "item.italian_delight.olive_door_item": "橄榄木门",
+  "item.italian_delight.pandoro_dough": "潘多酪面团",
+  "item.italian_delight.pandoro_slice": "潘多酪切片",
+  "item.italian_delight.pandoro": "潘多酪",
+  "item.italian_delight.panettone_slice": "潘娜托尼切片",
+  "item.italian_delight.panettone": "潘娜托尼",
+  "item.italian_delight.pasta_al_pesto": "青酱意面",
   "item.italian_delight.pizza_al_prosciutto": "生火腿披萨",
   "item.italian_delight.pizza_diavola": "魔鬼披萨",
   "item.italian_delight.pizza_diavola_slice": "魔鬼披萨切片",
   "item.italian_delight.pizza_margehrita": "玛格丽塔披萨",
   "item.italian_delight.pizza_margherita_slice": "玛格丽塔披萨切片",
   "item.italian_delight.pizza_prosciutto_slice": "火腿披萨切片",
+  "item.italian_delight.prewine_substance": "'拙'酒",
+  "item.italian_delight.raw_dante_special": "生但丁特制披萨",
   "item.italian_delight.raw_diavola": "生魔鬼披萨",
   "item.italian_delight.raw_margherita": "生玛格丽塔披萨",
   "item.italian_delight.raw_mushroom_pizza": "生蘑菇披萨",
   "item.italian_delight.risotto_al_sugo": "茄汁烩饭",
+  "item.italian_delight.rolling_pin": "擀面杖",
   "item.italian_delight.rosewine": "桃红葡萄酒",
-  "item.italian_delight.salami": "萨拉米",
-  "item.italian_delight.salami_pieces": "萨拉米片",
+  "item.italian_delight.salami": "萨拉米香肠",
+  "item.italian_delight.salami_pieces": "切片萨拉米香肠",
```

```suggestion
  "item.italian_delight.salami_pieces": "萨拉米香肠切片",
```
虽然原文和农夫乐事系列的蛋糕切片不同（）